src/utils/getFilteredPeople.tsx

```diff
@@ -0,0 +1,42 @@
+import { Person } from "../types";
+
+type Filters = {
+  sex: string | null;

```

Comment:
The `sex` filter is defined as `string | null`, but using `null` might not be ideal for filtering logic. Consider using an empty string or a default value that aligns with your filtering logic to avoid potential issues.